src/web/components/Counts.stories.tsx

```diff
@@ -41,13 +43,22 @@ export const Both = () => {
 
     return (
         <Container>
-            <Counts
-                ajaxUrl="https://api.nextgen.guardianapps.co.uk"
-                commentCount={239}
-                pageId="/lifeandstyle/2020/jan/25/deborah-orr-parents-jailers-i-loved"
-                pillar="news"
-                setIsExpanded={() => {}}
-            />
+            <Counts>
```

Comment:
I'm not too fussed, but feels like we could put `Counts` in meta now as it doesn't do a whole lot - it feels like it exists as a component for stories and we could move those stories to meta and keep the render tree shallower. I don't think for this PR though!

Reply:
I had the same though but, indeed, once you move it into meta then it's harder to test, you'd have to export it, or duplicate the border css, which felt messy so I held off.

I suppose a better solution would be to incorporate the counts tests into `ArticleMeta`